egs/chime4/s5_1ch/local/chain/tuning/run_tdnn_1d.sh

```diff
@@ -0,0 +1,346 @@
+#!/bin/bash
+
+# 1d is as 1c but introducing two non-splicing layers towards the beginning of
+#   the network.
+
+# local/chain/compare_wer.sh exp/chain/tdnn1c_sp exp/chain/tdnn1e3_sp
```

Comment:
You seem to have copied these comments from WSJ.  They report the results and diagnostics from WSJ.  You should replace these with suitable results and diagnostics from chime4.
In fact, you seem to have just copied the entire tuning directory from WSJ without really looking into its contents.  The idea is that when you commit the inital recipe for a setup you'll start with 1a.  Don't copy scripts that you haven't tested (e.g. the LSTM scripts).